.github/workflows/test-os.yml

```diff
@@ -0,0 +1,62 @@
+name: test-os
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  schedule:
+    - cron: '0 10 * * *'
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+      - 'docs/**'
+      - 'frontend/dockerfile/docs/**'
+
+env:
+  GO_VERSION: "1.19"
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          # - ubuntu-20.04
+          # - macOS-11
+          - windows-2022
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          cache: true
+      -
+        name: Go mod
+        run: |
+          go mod download
+      -
+        name: Test
```

Comment:
There's a few test jobs that don't use the reusable workflow - like this one. Is there a reason we can't use it in every case?

Reply:
Yeah unfortunately for `test-os` we can't pass the os (e.g. `windows-2022`) as an input to the reusable workflow. That's why I just moved this one in a dedicated wokrflow. I think GitHub is working on a fix for this.